packages/extension-browser/tests/end-to-end.ts

```diff
@@ -10,6 +10,22 @@ import { readFixture } from './helpers/fixtures';
 
 const pathToExtension = `${__dirname}/../bundle`;
 
+const wait = (delay = 500): Promise<void> => {
```

Comment:
Nit: We already have a `delay` in `utils/misc`.

Also you never call this with any parameter so might as well not have `delay` as one 😊

Reply:
I'll drop the param for now. Probably will wait on cutting over to the util until after David's PR is merged.

Reply:
Works for me!